proxy/http/HttpTunnel.cc

```diff
@@ -1161,6 +1165,8 @@ HttpTunnel::producer_handler(int event, HttpTunnelProducer *p)
     // If the write completes on the stack (as it can for http2), then
     // consumer could have called back by this point.  Must treat this as
     // a regular read complete (falling through to the following cases).
+    p->bytes_read = p->init_bytes_done;
+    // FALLTHROUGH
```

Comment:
Nitpick: If I feel like this comment is needed, I use `[[fallthrough]];`.

Reply:
Note - `[[fallthrough]]` is now official syntax - https://en.cppreference.com/w/cpp/language/attributes/fallthrough